ietf-te-types.yang

```diff
@@ -1249,9 +1253,9 @@ module ietf-te-types {
     description
       "Objective function for minimizing the cost on a path set.
       
-      This identity has been obsoleted: the
-      'svec-of-minimize-cost-path-set' identity SHOULD
-      be used instead.";
+       This identity has been obsoleted: the
```

Comment:
pls ensure the text is aligned in the file.. Also, is there a space at the beginning of the line?

Reply:
I have checked and the text says that the tiebreaker have been deprecated and these objective functions have been obsoleted

The space is coming from a Med's comment: see PR #279